### PR TITLE
Improve FTL tests

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/FTL/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:** 

## 10

---

Do not assume a query to google.com returns only one fixed AAAA record. Also, do not assume the IP address of ftl.pi-hole.net to be fixed for all time.

### CODE REVIEW

File: test/test_suite.bats [hunk 1]
```diff
@@ -145,14 +145,12 @@
...
-  [[ ${lines[1]} == "" ]]
 }
 
 @test "Known host is resolved as expected" {
-  run bash -c "dig ftl.pi-hole.net @127.0.0.1 +short"
+  run bash -c "dig A ftl.pi-hole.net @127.0.0.1 +short"
   printf "%s\n" "${lines[@]}"
-  [[ ${lines[0]} == "139.59.170.52" ]]
-  [[ ${lines[1]} == "" ]]
+  [[ ${lines[0]} != "0.0.0.0" ]]
```
Comment (dschaper) [inline test/test_suite.bats:153]: This will be true for any value that is not 0.0.0.0? Even empty or "`;; connection timed ;`"?

Comment (DL6ER) [reply test/test_suite.bats:153]: Yes, the test is not meant for successful DNS resolution but for not-blocked DNS resolution. However, we can leave the IP check for FTL as it is.

Comment (dschaper) [reply test/test_suite.bats:153]: It looks like `dig` failing completely will also end up as a green condition?

Comment (DL6ER) [reply test/test_suite.bats:153]: Yes

Comment (dschaper) [reply test/test_suite.bats:153]: If that's intended then I guess it does what it says on the tin.

Comment (DL6ER) [reply test/test_suite.bats:153]: You are right. I missed that. I will restore the previous test as it seems superior having this in mind.

**Pull request merged**